### PR TITLE
feat(backup, ark): change backup CRD names

### PR DESCRIPTION
**Pull request by mynktl**
**What this PR does / why we need it**:
This PR changes name of backup CRD:
  - `BackupCStor` CRD is changed to `CStorBackup` CRD
   - `BackupCStorLast` CRD is changed to `CStorCompletedBackup`

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: pkg/apis/openebs.io/v1alpha1/cstor_completed_backup.go [hunk 1]
```diff
@@ -23,22 +23,22 @@ import (
 // +genclient
 // +genclient:noStatus
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +resource:path=backupcstorlast
+// +resource:path=cstorcompletedbackup
 
-// BackupCStorLast describes a cstor last-backup resource created as custom resource
-type BackupCStorLast struct {
+// CStorCompletedBackup describes a cstor completed-backup resource created as custom resource
+type CStorCompletedBackup struct {
```
Comment (AmitKumarDas) [inline pkg/apis/openebs.io/v1alpha1/cstor_completed_backup.go]: Can there be multiple completed backups?
Does it make sense to have `CStorLastCompletedBackup`

Comment (kmova) [reply pkg/apis/openebs.io/v1alpha1/cstor_completed_backup.go]: Is this only going to store the recently completed backup? Then CStorRecentlyCompletedBackup makes sense.

File: pkg/install/v1alpha1/openebs_crds.go [hunk 2]
```diff
@@ -323,18 +323,18 @@ spec:
...
-    plural: backupcstorlasts
-    singular: backupcstorlast
-    kind: BackupCStorLast
+    plural: cstorcompletedbackups
+    singular: cstorcompletedbackup
+    kind: CStorCompletedBackup
     shortNames:
-    - bkplast
-    - backuplast
+    - bkpcompleted
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/openebs_crds.go]: can it be `cbkpc`

File: pkg/install/v1alpha1/openebs_crds.go [hunk 3]
```diff
@@ -291,15 +291,15 @@ spec:
...
   scope: Namespaced
   names:
-    plural: backupcstors
-    singular: backupcstor
-    kind: BackupCStor
+    plural: cstorbackups
+    singular: cstorbackup
+    kind: CStorBackup
     shortNames:
     - bkp
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/openebs_crds.go]: can this be `cbkp`

File: pkg/install/v1alpha1/openebs_crds.go (hunk 2 above)
Comment (mynktl) [reply pkg/install/v1alpha1/openebs_crds.go]: ok

File: pkg/install/v1alpha1/openebs_crds.go (hunk 3 above)
Comment (mynktl) [reply pkg/install/v1alpha1/openebs_crds.go]: ok. I will add prefix `c` to short names to avoid any conflict with velero/ark CRD.

**Pull request merged**